docs/installing-packages.md

```diff
@@ -0,0 +1,203 @@
+# Installing packages
+This documentation dives deeper into how a user can configure packages for installation inside the EIB image. Furthermore, it explains the **RPM resolution** process that EIB goes through so it can ensure that configured packages will be successfully installed even in an **air-gapped** environment.
+
+Below you can find a table of contents for easier navigation:
```

Comment:
This is arguably not needed. Go to https://github.com/suse-edge/edge-image-builder/blob/main/docs/building-images.md and click the button on the right that has three rows of a dot and a line. GitHub automatically generates a TOC based on the headings.